packages/editor-ui/src/components/canvas/elements/edges/CanvasEdge.vue

```diff
@@ -36,5 +80,25 @@ const path = computed(() =>
 		:label-bg-style="{ fill: 'red' }"
 		:label-bg-padding="[2, 4]"
 		:label-bg-border-radius="2"
+		tabindex="2"
 	/>
+	<EdgeLabelRenderer>
```

Comment:
Is it necessary to have multiple template roots here? If yes, can we use [fragments](https://v3-migration.vuejs.org/new/fragments) for this? Otherwise, we can just wrap them in a container

Reply:
That's what we're doing here. There's no explicit syntax for fragments in Vue(like `<>..</>` in React). It's just ESlint not "realizing" this is Vue 3 project. So, I wouldn't add another container as those would add up quickly for each edge.

Reply:
Ah, I thought we are not handling the props correctly or something and that's why `eslint` is complaining. Does it make sense to turn off this rule for `editor-ui`?